### PR TITLE
Updated markdown.js

### PR DESCRIPTION
**Pull request by amruta-kashikar**
Added social links

### CODE REVIEW

File: src/components/markdown.js [hunk 1]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
                 username={props.social.medium} /></>
             <><DisplaySocial base='https://www.youtube.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/youtube.svg'
                 username={props.social.youtube} /></>
+            <><DisplaySocial base='https://www.codechef.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.1.0/icons/codechef.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:143]: Replace `base='https://www.codechef.com/c'` with `base='https://www.codechef.com/'` to make this work properly.

File: src/components/markdown.js [hunk 2]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.codechef} /></>
+            <><DisplaySocial base='https://www.hackerrank.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/hackerrank.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:145]: remove `/c` from the base prop

File: src/components/markdown.js [hunk 3]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.hackerrank} /></>
+            <><DisplaySocial base='https://www.codeforces.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/codeforces.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:147]: remove `/c` from the base prop

File: src/components/markdown.js [hunk 4]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.codeforces} /></>
+            <><DisplaySocial base='https://www.leetcode.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/leetcode.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:149]: remove `/c` from the base prop

File: src/components/markdown.js [hunk 5]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.leetcode} /></>
+            <><DisplaySocial base='https://www.hackerearth.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/hackerearth.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:151]: remove `/c` from the base prop

File: src/components/markdown.js [hunk 6]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.hackerearth} /></>
+            <><DisplaySocial base='https://www.geeksforgeeks.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/geeksforgeeks.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:153]: remove `/c` from the base prop

File: src/components/markdown.js [hunk 7]
```diff
@@ -140,9 +140,23 @@ const Markdown = (props) => {
...
+                username={props.social.geeks_for_geeks} /></>
+            <><DisplaySocial base='https://www.topcoder.com/c' icon='https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/topcoder.svg'
```
Comment (rahuldkjain) [inline src/components/markdown.js:155]: remove `/c` from the base prop

**Review by rahuldkjain — CHANGES_REQUESTED** [review]

**Comment by amruta-kashikar** [comment]
`/c` removed

**Comment by amruta-kashikar** [comment]
Changes done

**Review by rahuldkjain — APPROVED** [review]
Great @amruta-kashikar 💯

**Pull request merged**